OverlayPlugin.Core/EventSources/EnmityEventSource.cs

```diff
@@ -203,6 +203,8 @@ protected override void Update()
 
                 var combatants = memory.GetCombatantList();
 
+                combatants.RemoveAll((c) => c.Type != ObjectType.PC && c.Type != ObjectType.Monster);
```

Comment:
That filter only excludes combatants from the Enmity list. I specifically moved it out of EnmityMemoryxx.cs so that `getCombatants` calls will return all combatants regardless of type.

Reply:
oh,i konw

Reply:
If there were a toggle, it would probably belong on the OP WS getCombatants call, so that the client can choose the current filtered behavior, or the new unfiltered behavior.